tests/README.md

```diff
@@ -185,6 +185,29 @@ For executing this test, first make sure you are in the correct environment as d
 pytest tests/unit/test_notebooks_python.py::test_sar_single_node_runs
 ```
 
+### Test execution with tox
```

Comment:
One question, when we deprecate the ADO pipeline, are we going to use tox as the main test tool instead of pytest?
If that's the case, above in the text we have content showing the behavior with pytest:
`pytest tests/smoke -m "smoke and not spark and gpu" --durations 0` will we still need this?

Reply:
I was hoping people can use `tox` as another option (especially as a way to debug when they encounter build problems in the pipeline), I want to keep the option here for people to use either pytest or tox. I myself runs pytest most of the time due to habit 😄. So I think we can keep these nice documentations on pytest.